packages/experimental/TabList/src/useTab.ts

```diff
@@ -8,8 +14,66 @@ import type { TabProps, TabInfo } from './Tab.types';
  * @returns configured props and state for TabsItem
  */
 export const useTab = (props: TabProps): TabInfo => {
+  const defaultComponentRef = React.useRef<IFocusable>(null);
+  const { accessibilityLabel, accessible, componentRef = defaultComponentRef, tabKey, disabled, icon, ...rest } = props;
+  // Grabs the context information from Tabs (currently selected TabsItem and client's onTabsClick callback).
+  const info = React.useContext(TabListContext);
+
+  const changeSelection = React.useCallback(() => {
+    if (tabKey != info.selectedKey) {
+      info.onTabSelect && info.onTabSelect(tabKey);
+      info.updateSelectedTabRef && componentRef && info.updateSelectedTabRef(componentRef);
+    }
+  }, [componentRef, info, tabKey]);
+
+  const changeSelectionWithFocus = useOnPressWithFocus(componentRef, changeSelection);
+
+  const pressable = usePressableState({
+    ...rest,
+    onPress: changeSelectionWithFocus,
+    onFocus: changeSelection,
+  });
+
+  const onKeyUpProps = useKeyProps(changeSelection, ' ', 'Enter');
+
+  // Used when creating accessibility properties in mergeSettings below.
+  const onAccessibilityAction = React.useCallback(
+    (event: { nativeEvent: { actionName: any } }) => {
+      switch (event.nativeEvent.actionName) {
+        case 'Select':
+          changeSelection();
+          break;
+      }
+    },
+    [changeSelection],
+  );
+
+  /* We use the componentRef of the currently selected tabsItem to maintain the default tabbable
+  element in Tabs. Since the componentRef isn't generated until after initial render,
+  we must update it once here. */
+  React.useEffect(() => {
+    if (tabKey == info.selectedKey) {
+      info.updateSelectedTabRef && componentRef && info.updateSelectedTabRef(componentRef);
+    }
+  }, [componentRef, info, tabKey]);
+
   return {
-    props: props,
-    state: {},
+    props: {
+      ...pressable.props,
+      accessible: accessible ?? true,
+      accessibilityRole: 'tab',
+      focusable: !disabled ?? true,
+      accessibilityState: { disabled: disabled, selected: info.selectedKey === tabKey },
```

Comment:
Can we use the memoized getAccessibilityStateWorker function for this like we do for other controls (see [useMenuItem.ts](https://github.com/microsoft/fluentui-react-native/blob/d1fc0c774448d103085b5bb5ab0103678a62cbbc/packages/components/Menu/src/MenuItem/useMenuItem.ts#L129))? That way, someone can pass in accessibilityState if they want to and override these values.